cves/CVE-2017-9996.yml

```diff
@@ -129,7 +134,13 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    The file containing the vulnerability, cdxl.c, saw a fair amount of
+    development through the lifetime of the vulnerability. There was about
+    six developers who regularly worked on the file and mostly worked on bug
+    fixes and improving the code. One thing that was interesting about this 
+    vulnerability was that the line that the vulnerability was on was edited
+    between when the vulnerability was initially put into the code.
```

Comment:
I am struggling to understand what you are trying to say with the last sentence here, it is just a bit too wordy for clear understanding. Are you saying that they changed the line that caused the vulnerability within the lifetime of it, but they didn't actually fix the vulnerability?